src/Compilers/Core/Portable/CommandLine/AnalyzerConfig.cs

```diff
@@ -110,7 +110,7 @@ public static AnalyzerConfig Parse(string text, string pathToFile)
         /// </summary>
         public static AnalyzerConfig Parse(SourceText text, string pathToFile)
         {
-            if (!Path.IsPathRooted(pathToFile) || string.IsNullOrEmpty(Path.GetFileName(pathToFile)))
+            if (!Path.IsPathFullyQualified(pathToFile) || string.IsNullOrEmpty(Path.GetFileName(pathToFile)))
```

Comment:
Should the workspaces API also be having an identical check to prevent AnalyzerConfigDocuments from being added? Otherwise it means you can create a project that will blow up the moment you try adding a tree.

Reply:
Up to you, but sounds reasonable to me

Reply:
I guess if we *do* do it, then we we have to duplicate the check somewhere. Do we just copy/paste the logic?

Thinking further I'm also not sure what it means if we have non-C#/VB users of the workspace, and whether they'd want the restriction or not. I guess we still would use our parser for them which is maybe the right or wrong thing...not sure.